src/administration/integrations/webhooks.md

```diff
@@ -1,17 +1,97 @@
 # Generic Webhook
 
-This hook will allow you to capture any push events on platform and POST
-a JSON file describing the activity to the url of your choice. You can
-use this to further automate your Platform.sh workflow. 
-
-For example the `routes` object keys will give you urls against which you can 
-launch automated user testing.
+This hook will allow you to capture any push events on platform and POST a JSON file describing the activity to the url of your choice. You can use this to further automate your Platform.sh workflow.
 
 ```bash
 $ platform integration:add --type=webhook --url=A-URL-THAT-CAN-RECEIVE-THE-POSTED-JSON
 ```
 
-Here you can see an example of the response:
+The webhook URL will receive a POST message for every "Activity" that is triggered, and the message will contain complete information about the entire state of the project at that time.  In practice most of the message can be ignored but is available if needed.  The most commonly used values are documented below.
+
+It's also possible to set the integration to only send certain activity types, or only activities on certain branches.  The CLI will prompt you to specify which to include or exclude.  Leave at the default values to get all events on all environments in a project.
+
+## Webhook schema
+
+### `id`
+
+A unique opaque value to identify the activity.
+
+### `project`
+
+The Project ID for which the activity was triggered.  Use this value if you want to have multiple projects POST to the same URL.
+
+### `type`
+
+The `type` property specifies the event that happened.  Its value is one of:
+
+* `project.modify.title`: The human-friendly title of the project has been changed.
+* `project.create`: A project has been created.  Although it will appear in the activity feed exactly once, it will not be sent via a webhook as it will always happen before a webhook can be configured.
```

Comment:
Its correct that this will never be sent to the webhook. But it goes against the claim that "type has one of the following values". Maybe not list it at all?

Reply:
Discussed a bit in channel.  Going to leave in for completeness, since it's described as being in the feed but not the webhook.